.github/workflows/depaware.yml

```diff
@@ -26,3 +26,6 @@ jobs:
 
     - name: depaware tailscale
       run: go run github.com/tailscale/depaware --check tailscale.com/cmd/tailscale
+
+    - name: depaware derper
+      run: go run github.com/tailscale/depaware --check tailscale.com/cmd/derper
```

Comment:
I think all three of these can be combined into one check and then it's more efficient, sharing state?